components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1697,10 +1734,29 @@ public void deleteGroup(String groupId) throws NotFoundException, CharonExceptio
     @Override
     public List<Object> listGroupsWithGET(Node rootNode, int startIndex, int count, String sortBy, String sortOrder,
             String domainName, Map<String, Boolean> requiredAttributes)
+
             throws CharonException, NotImplementedException, BadRequestException {
         if (sortBy != null || sortOrder != null) {
```

Comment:
Move above new line to here, where the method starts

Reply:
Fixed: 2aa3adf18f5d3517e09c218c2c8be0eb29ac6e4e